src/theme/style-overrides/components.ts

```diff
@@ -0,0 +1,16 @@
+import { ThemeOptions } from "@mui/material";
+
+const COMPONENTS_STYLE_OVERRIDES: ThemeOptions["components"] = {
+  MuiButton: {
+    styleOverrides: {
+      root: {
+        paddingInline: "16px",
+      },
+      outlinedPrimary: ({ theme }) => ({
+        border: `1px solid ${theme.palette.primary.main}`,
+      }),
+    },
+  },
+};
```

Comment:
Idea (not important now): Perhaps [spacing controls](https://mui.com/material-ui/customization/spacing/) could work?